src/webgpu/capability_info.ts

```diff
@@ -290,6 +290,17 @@ export function depthStencilFormatAspectSize(
   return texelAspectSize;
 }
 
+export function dimensionTypeAndFormatCombinationError(
```

Comment:
uNit: Prefer to invert this as that better describes "capability info" per the filename: true if the combination is valid, false if error. e.g. "textureDimensionAndFormatCompatible"

Reply:
Done